agent/runner/jobs/mongodb_restore_job.go

```diff
@@ -136,7 +137,6 @@ func (j *MongoDBRestoreJob) Run(ctx context.Context, send Send) error {
 	go func() {
 		err := j.jobLogger.streamLogs(streamCtx, send, restoreOut.Name)
 		if err != nil && !errors.Is(err, io.EOF) && !errors.Is(err, context.Canceled) {
-			j.jobLogger.sendLog(send, err.Error(), false)
```

Comment:
Do we really need remove this line?

Reply:
yeah, we currently try to send only pbm logs to job logs - this breaks that. Also, if stream logs failed, it makes sense to log directly (instead of attempting to send another log to the PMM server which might also fail).

Reply:
My concern is: will it be tested in the scope of these changes?

Reply:
Can you clarify what you mean by "tested" here? If you mean tested by QA, then we could inform them to check logs again